template/setup/index.js

```diff
@@ -1,14 +1,32 @@
 const fs = require('fs');
 const path = require('path');
 const { execSync } = require('child_process');
-const packageJson = require('../package.json');
 const { prompt } = require('enquirer');
 
 async function setup() {
   const deleteFile = fileName => fs.unlinkSync(path.join(__dirname, fileName));
   const deleteDirectory = dirName => fs.rmdirSync(path.join(__dirname, dirName));
   const writeFile = (fileName, data) => fs.writeFileSync(path.join(__dirname, fileName), data);
 
+  console.log('\n📝  Configuring project display name and bundle identifier...');
+  const fs = require('fs');
+  const path = require('path');
+  const { execSync } = require('child_process');
+  const { prompt } = require('enquirer');
+
+  async function setup() {
+    const deleteFile = fileName => fs.unlinkSync(path.join(__dirname, fileName));
+    const deleteDirectory = dirName => fs.rmdirSync(path.join(__dirname, dirName));
+    const writeFile = (fileName, data) => fs.writeFileSync(path.join(__dirname, fileName), data);
```

Comment:
this looks like a merge error? we are defining this and the `require`'s twice

Reply:
yup, silly error on my part